share/spice/cryptocurrency/cryptocurrency.js

```diff
@@ -101,12 +101,15 @@
                 Converter.fromCurrency = from;
             }
 
-            var endpoint = "https://api.cryptonator.com/api/full/" + from + "-" + to;
-            $.get(endpoint, function(payload) {
-                Converter.rate = parseFloat(payload.ticker.price);
+            var endpoint = "/js/spice/cryptocurrency/cryptonator/" + from + "/" + to;
```

Comment:
Should be `"/js/spice/cryptonator/"`

Its as if we're creating a separate IA -- its not a subpath

I would name it `cryptocurrency_json`